src/components/sidebar/PageButton.tsx

```diff
@@ -23,8 +24,8 @@ export default function PageButton({
   const buttonStyles = `${
     isOpen
       ? "w-[14.375rem] flex justify-start pl-6"
-      : "w-[3.125rem] tooltip hover:tooltip-open tooltip-right before:bg-base-100 before:text-base-300 after:border-r-base-100 overflow:false"
-  } h-[3.125rem] text-base-300 capitalize border-none hover:bg-base-100`;
+      : "w-[3.125rem] tooltip hover:tooltip-open tooltip-right overflow:false px-0"
```

Comment:
You can get rid of these styles

tooltip hover:tooltip-open tooltip-right overflow:false